receiver/hostmetricsreceiver/internal/scraper/diskscraper/factory_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package diskscraper
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+)
+
+func TestCreateMetricsScraper(t *testing.T) {
+	factory := &Factory{}
+	cfg := &Config{}
+
+	scraper, err := factory.CreateMetricsScraper(context.Background(), zap.NewNop(), cfg, nil)
+
+	assert.Nil(t, err)
```

Comment:
Assert.NoError please :) Maybe a separate PR and all the code you wrote :)

Reply:
Yea I'll do another cleanup PR shortly

Reply:
There's 88 references of `assert.Nil(t, err)` in the repo btw - should I change them all in a follow-up PR?